packages/components/src/components/accordion/accordion-styles.ts

```diff
@@ -28,19 +28,25 @@ export const getComponentCss = (
   size: BreakpointCustomizable<AccordionSize>,
   compact: boolean,
   open: boolean,
-  theme: Theme
+  theme: Theme,
+  sticky: boolean
 ): string => {
-  const { primaryColor, hoverColor, contrastLowColor } = getThemedColors(theme);
+  const { primaryColor, hoverColor, contrastLowColor, backgroundColor } = getThemedColors(theme);
   const {
     primaryColor: primaryColorDark,
     hoverColor: hoverColorDark,
     contrastLowColor: contrastLowColorDark,
+    backgroundColor: backgroundColorDark,
   } = getThemedColors('dark');
+  const zIndexes = ['collapsible', 'heading'];
 
   return getCss({
     '@global': {
       ':host': {
         display: 'block',
+        ...(sticky && {
+          position: 'relative',
```

Comment:
a comment would be helpful why a `position` is needed.

Reply:
Actually I looked it up and it's not needed since position sticky tells the parent already to be relative, so this is not needed. Thanks!